index.html

```diff
@@ -1636,8 +1636,18 @@ <h2>Definition of Roles</h2>
 		<div class="role" id="caption">
 			<rdef>caption</rdef>
 			<div class="role-description">
-				<p>On-screen descriptive text for a figure or table in the page.</p>
-				<p>Authors SHOULD reference the element with role <code>caption</code> by setting <pref>aria-describedby</pref> on the figure or table.</p>
+				<p>A visible name or caption for an image, table or grid roles.  See synonym <rref>legend</rref>.</p>
+				<p>An element with role <code>caption</code> MAY provide an accessible name for  <a href="#namefromcaptioncaption">elements that support the "caption/legend" naming technique</a> by using one of three methods:</p>
+				<ul>
+					<li>caption: The element with role <code>caption</code> is the first child of the element it is naming.</li>
+					<li>caption: The element with role <code>caption</code> is the last child of the element it is naming.</li>
+					<li>reference: The element with role <code>caption</code> is referenced by the element it names via the <pref>aria-labelledby</pref> attribute.</li>
```

Comment:
mention `aria-owns` to keep the caption/legend associated with the element it names?  (more on this in comments further down the document)

Reply:
resolving per previous comment about required context roles